app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      include Wor::Paginate
+
+      # GET /api/v1/:user_id/rents
+      def index
+        render_paginated rents
+      rescue ActiveRecord::RecordNotFound => e
```

Comment:
we can put this rescue on `ApiController`  because we found it on `BookController` too. We use `rescue_from` https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from